### PR TITLE
Create classes

### PR DESCRIPTION
**Pull request by nevisende**
### Person class includes:
  - Instance vars: `@id`, `@name`, and `@age`.
  - Constructor with `name`,  `age`, and `parent_permission` as parameter. `name` and `parent_permission` are optional and have default values of `"Unknown"` and `true`.
  - Getters for `@id`, `@name`, and `@age`.
  - Setters for `@name` and `@age`.
  - Private method `is_of_age?` that returns `true` if `@age` is greater or equal to 18 and `false` otherwise.
  - Public method `can_use_services?` that returns `true` if a person is of age or if they have permission from parents.
### Student class includes:
  - Inherits from `Person`.
  - The constructor extends the parent's constructor by adding `@classroom` and a parameter for it.
  - Method `play_hooky` that returns `"¯\(ツ)/¯"`.
### Teacher class includes:
  - Inherits from `Person`.
  - The constructor extends the parent's constructor by adding `@specialization` and a parameter for it.
  - Override `can_use_services?` so it always returns `true`.

### CODE REVIEW

**Review by eri8-9 — COMMENTED** [review]
Hi @nevisende, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

wrong way

## Invalid Code Review Request

You have submitted a project, that has Linters Errors and does not include a descriptive readme file.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: person.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  end
+
+  def of_age?
+    @age >= 18
+  end
+  private :is_of_age?
+
+  def can_use_services?
+    is_of_age? || @parent_permission
+  end
```
Comment (eri8-9) [inline person.rb:19]: - [ ] You have declared a method called "of_age?" and you've been using it as "is_of_age?" that eventually doesn't exist. Please make it work. 😉

**Review by eri8-9 — CHANGES_REQUESTED** [review]
Hi @nevisende, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

almost there

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -1 +1,40 @@
...
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Setup
+
+- Simply clone the project (for more information please check the link https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository
+- Run the cmd promt
+- Type "build i" for the Gemfile.lock
```
Comment (elisha2kyakpo1) [inline README.md:19]: [optional] 
- [ ] Let's have a correct command for the Ruby dependencies.
- [ ] It's also a good idea to write down all the necessary commands required for running or getting the project. For example, how to clone a repo.

**Review by elisha2kyakpo1 — APPROVED** [review]
Hi @nevisende, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**